LittleDatabaseApp/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using LittleDatabaseApp.Enums;
+using SubjectsDAL.EF;
+using SubjectsDAL.Models;
+using SubjectsDAL.Repos;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LittleDatabaseApp
+{
+    class Program
+    {
+        public static bool exitApp;
+
+        public static List<Human> humans = new List<Human>();
+        public static List<Android> androids = new List<Android>();
+
+        static void Main(string[] args)
+        {
+            Database.SetInitializer(new DataInitializer());
+
+            while (!exitApp)
+            {
+                DisplayMenu();
+            }
+        }
+
+        private static void DisplayMenu()
+        {
+            Console.WriteLine($"Choose option\n1) Add Person\n2) Show peoples in memory\n3) Add people to database\n4) Show database records\n5) Exit App");
+
+            string userInput = Console.ReadLine();
+
+            if (userInput == "5")
+            {
+                exitApp = true;
+                return;
+            }
+
+            RecognizeUserInput(userInput);
+
+            Console.ReadLine();
+            Console.Clear();
+        }
+
+        private static void RecognizeUserInput(string userInput)
+        {
+            MenuOptions MenuChoosen = new MenuOptions();
```

Comment:
Please consider avoiding strong typing + try to name variables as its type for example you have `MenuOptions` type so the variable should be called `menuOptions`